html/webappapis/system-state-and-capabilities/the-navigator-object/protocol-handler-fragment-manual.https.html

```diff
@@ -0,0 +1,18 @@
+<!doctype html>
+<meta charset=windows-1254>
```

Comment:
If this is intentional it'd be good to add a comment stating so.

Reply:
Yeah, it's to test the `registerProtocolHandler()` call.